src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -10,9 +10,16 @@ interface PaymentSubmitButtonTextProps {
     methodGateway?: string;
     methodId?: string;
     methodType?: string;
+    continueWith?: string;
```

Comment:
should this be `paymentMethodName` rather than `continueWith` ?

Reply:
I think I'm mixing existing props that pass only data, with a new prop which is trying to portray behaviour. 

Would we be happier with something more like: https://github.com/bigcommerce/checkout-js/pull/598 ?

Reply:
Moved to: https://github.com/bigcommerce/checkout-js/pull/598